### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by Redhawk18**
neither of us want annoying questions anymore

### CODE REVIEW

File: src/main/java/baritone/behavior/MapBuilderBehavior.java [hunk 1]
```diff
@@ -476,6 +501,9 @@ public void onTick(TickEvent event) {
                     return;
                 }
 
+
+
+                //never used loop
```
Comment (mankool0) [inline src/main/java/baritone/behavior/MapBuilderBehavior.java:506]:
```suggestion
                // Loot shulker and update it's contents
```

File: src/main/java/baritone/process/BuilderProcess.java [hunk 2]
```diff
@@ -631,6 +631,7 @@ private Goal assemble(BuilderCalculationContext bcc, List<IBlockState> approxPla
     }
 
     private Goal assemble(BuilderCalculationContext bcc, List<IBlockState> approxPlaceable, boolean logMissing) {
+        int debugvar = 0;
```
Comment (mankool0) [inline src/main/java/baritone/process/BuilderProcess.java:634]: What for?

File: src/main/java/baritone/process/BuilderProcess.java [hunk 3]
```diff
@@ -679,6 +680,8 @@ private Goal assemble(BuilderCalculationContext bcc, List<IBlockState> approxPla
                 logDirect(missing.entrySet().stream()
                         .map(e -> String.format("%sx %s", e.getValue(), e.getKey()))
                         .collect(Collectors.joining("\n")));
+                debugvar++;
```
Comment (mankool0) [inline src/main/java/baritone/process/BuilderProcess.java:683]: What for?

File: src/main/java/baritone/process/BuilderProcess.java [hunk 4]
```diff
@@ -679,6 +680,8 @@ private Goal assemble(BuilderCalculationContext bcc, List<IBlockState> approxPla
...
+                logDebug("debugvar = " + debugvar);
```
Comment (mankool0) [inline src/main/java/baritone/process/BuilderProcess.java:684]: What for?

**Review by mankool0 — CHANGES_REQUESTED** [review]
Please fix :)

File: src/main/java/baritone/process/BuilderProcess.java (hunk 2 above)
Comment (Redhawk18) [reply src/main/java/baritone/process/BuilderProcess.java:634]: we dont talk about this

File: src/main/java/baritone/process/BuilderProcess.java (hunk 3 above)
Comment (Redhawk18) [reply src/main/java/baritone/process/BuilderProcess.java:683]: when i thought this was an issue

**Pull request merged**